plugins/reporters/spdx/src/main/kotlin/SpdxDocumentModelMapper.kt

```diff
@@ -70,7 +70,11 @@ internal object SpdxDocumentModelMapper : Logging {
                 ortResult
             )
 
-            ortResult.getDependencies(project.id, 1).mapTo(relationships) { dependency ->
+            ortResult.getDependencies(
+                id = project.id,
+                maxLevel = 1,
+                omitExcluded = false
```

Comment:
At this commit, the `omitExcluded` parameter does not exist yet and should be removed here and below.